From 23c649309399b49456c5eaf10060fdc77a123bfb Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Fri, 5 Feb 2010 14:21:48 -0500
Subject: [PATCH] Move multipath scan earlier.  It must go before any other dm
 rules.

multipath scan needs to go before lvm, or else lvm may grab the device
multipath would activate, and multipath will be unable to claim it.
---
 modules.d/90multipath/{14-multipath-scan.rules => 9-multipath-scan.rules} | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 rename modules.d/90multipath/{14-multipath-scan.rules => 9-multipath-scan.rules} (100%)

diff --git a/modules.d/90multipath/14-multipath-scan.rules b/modules.d/90multipath/9-multipath-scan.rules
similarity index 100%
rename from modules.d/90multipath/14-multipath-scan.rules
rename to modules.d/90multipath/9-multipath-scan.rules
